data-prepper-plugins/rss-source/build.gradle

```diff
@@ -18,6 +18,7 @@ dependencies {
     implementation 'com.fasterxml.jackson.core:jackson-core'
     implementation 'com.fasterxml.jackson.core:jackson-databind'
     implementation 'com.apptasticsoftware:rssreader:3.2.5'
+    implementation 'com.google.code.gson:gson:2.8.7' // TODO: Pretty print Items for demo, remove later
```

Comment:
This should be removed now, right?

Reply:
I'm keeping it for future intermediate demos.

Reply:
I think we should remove it now. A couple reasons:

1. You can use Jackson for printing demo data
2. I don't even think it is being used
3. We might forget to remove it

Reply:
It was being used in the `RSSSource.java` while creating the documents, I was printing them using GSON. But I'll look at other alternatives.